sweetest/sweetest-core/src/main/java/com/mysugr/sweetest/framework/dependency/DependencyMode.kt

```diff
@@ -16,15 +14,15 @@ enum class DependencyMode {
     // configured from new `provide` or old `requireReal` or `realOnly`
 
     /**
-     * Dependency that is provided with a custom initializer with [BaseBuilder.provide].
```

Comment:
Why remove `[…]`? I think it's nice to references in KDoc navigatable.

Reply:
it didn't resolve in the IDE, therefore i removed it